workinggroups.md

```diff
@@ -0,0 +1,14 @@
+# Open Data Cube Working Groups
```

Comment:
Perhaps add a sentence explaining the general functioning of working groups -- i.e. that they focus on a specific element of the ODC, they report directly to the steering council, and that membership is voluntary. Maybe also mention that they can contact the current chair if they want to join?

I'd also suggest adding a bullet point list of the different working groups, in case we add more in future.

Reply:
Worked to address your comments in new draft.